bigbluebutton-tests/playwright/parameters/customparameters.js

```diff
@@ -547,6 +547,57 @@ class CustomParameters extends MultiUsers {
 
     await checkScreenshots(this, 'should be on custom layout', [e.webcamContainer, e.webcamMirroredVideoContainer], 'enforce-custom-layout', 4);
   }
+
+  async predefinedGroups() {
+    await this.modPage.waitForSelector(e.whiteboard);
+    await this.userPage.waitForSelector(e.whiteboard);
+    await this.userPage2.waitForSelector(e.whiteboard);
+
+    await this.modPage.waitAndClick(e.manageUsers);
+    await this.modPage.waitAndClick(e.createBreakoutRooms);
+
+    await this.modPage.hasElement(e.breakoutBox0, 'should display the breakout box for unassigned users,');
+
+    await this.modPage.hasElement(e.breakoutBox1, 'should display the breakout box for room 1');
+    await this.modPage.hasElementCount(`${e.breakoutBox1} > p`, 1, 'should display only 1 user in the breakout room 1');
+    await this.modPage.hasText(e.breakoutBox1, this.userPage.username, 'should display the correct user assigned to breakout room 1');
+    await this.modPage.hasValue(e.roomNameInput, 'Room 1', 'should display the correct name of the group for breakout room 1');
+
+    await this.modPage.hasElement(e.breakoutBox2, 'should display the breakout box for room 2');
+    await this.modPage.hasElementCount(`${e.breakoutBox2} > p`, 1, 'should display only 1 user in the breakout room 2');
+    await this.modPage.hasText(e.breakoutBox2, this.userPage2.username, 'should display the correct user assigned to breakout room 2');
+
+    await this.modPage.hasValue(`input[data-test=roomName-2]`, 'Room 2', 'should display the correct name of the group for breakout room 2');
```

Comment:
`e.roomNameInput` refers to roomName-1. I'd suggest updating this variable name to `e.roomNameInput1` and creating the `e.roomNameInput2`. Avoid using hardcoded selectors